framework/wcf/Extensibility/Transport/AzureQueueStorageClient/AzureQueueStorageClient/SampleProfileAzureQueueStorageBinding.cs

```diff
@@ -0,0 +1,434 @@
+            ApplyConfiguration(configurationName);
+        }*/
+
+        public bool OrderedSession
+        {
+            get { return session.Ordered; }
+            set { session.Ordered = value; }
+        }
+
+        public bool ReliableSessionEnabled
+        {
+            get { return reliableSessionEnabled; }
+            set { reliableSessionEnabled = value; }
+        }
+
+        public override string Scheme { get { return "soap.udp"; } }
+
+        public TimeSpan SessionInactivityTimeout
+        {
+            get { return this.session.InactivityTimeout; }
+            set { this.session.InactivityTimeout = value; }
+        }
+
+        public EnvelopeVersion SoapVersion
+        {
+            get { return EnvelopeVersion.Soap12; }
+        }
+
+        /*public Uri ClientBaseAddress
+        {
+            get { return this.compositeDuplex.ClientBaseAddress; }
+            set { this.compositeDuplex.ClientBaseAddress = value; }
+        }*/
+
+        /// <summary>
+        /// Create the set of binding elements that make up this binding. 
+        /// NOTE: order of binding elements is important.
+        /// </summary>
+        /// <returns></returns>
+        public override BindingElementCollection CreateBindingElements()
+        {   
+            BindingElementCollection bindingElements = new BindingElementCollection();
+
+            if (ReliableSessionEnabled)
+            {
+                bindingElements.Add(session);
+                //bindingElements.Add(compositeDuplex);
+            }
+
+            bindingElements.Add(encoding);
+            bindingElements.Add(transport);
+
+            return bindingElements.Clone();
+        }
+
+       /* void ApplyConfiguration(string configurationName)
+        {
+            SampleProfileUdpBindingCollectionElement section = (SampleProfileUdpBindingCollectionElement)ConfigurationManager.GetSection(UdpConstants.UdpBindingSectionName);
+            SampleProfileUdpBindingConfigurationElement element = section.Bindings[configurationName];
+            if (element == null)
+            {
+                throw new ConfigurationErrorsException(string.Format(System.Globalization.CultureInfo.CurrentCulture,
+                    "There is no binding named {0} at {1}.", configurationName, section.BindingName));
+            }
+            else
+            {
+                element.ApplyConfiguration(this);
+            }
+        }*/
+
+        void Initialize()
+        {
+            transport = new AzureQueueStorageTransportBindingElement();
+            session = new ReliableSessionBindingElement();
+            //compositeDuplex = new CompositeDuplexBindingElement();
+            encoding = new TextMessageEncodingBindingElement();
+        }
+
+        //initialize a SampleProfileUdpBinding from the info collected in a ReliableSessionBindingElement if one is present.
+        void InitializeFrom(AzureQueueStorageTransportBindingElement udpTransportBindingElement,
```

Comment:
Remove this method and all methods below it, including the commented out ones. They are only needed when we have the ability to create custom binding elements from a WSDL, and we don't have that yet and won't for quite a while.

Reply:
i see. Cleaned up this section,